src/components/ChatApp/Settings/Settings.css

```diff
@@ -1,5 +1,5 @@
 .tabStyle{
-  height: 550px;
+  height: auto;
```

Comment:
could you please explain this ?

Reply:
This change was made in a commit for #1017 . I branched into button-patch at this point, so this change came in here as well. For that particular issue, this change makes the left tab in settings page adjust the height according to the amount of content to be displayed.
8 tabs when looged out, hence 400px; 11 tabs when logged in, hence 550px.